IPython/terminal/shortcuts/auto_suggest.py

```diff
@@ -350,7 +351,7 @@ async def error_catcher(buffer):
             try:
                 await self._trigger_llm_core(buffer)
             except Exception as e:
-                get_ipython().log.error("error")
+                get_ipython().log.error(f"error {e}")
```

Comment:
You should never format strings in logging: https://docs.astral.sh/ruff/rules/logging-percent-format/, in addition formatting (for %) prevent structured logging.